full/src/test/java/apoc/couchbase/CouchbaseIT.java

```diff
@@ -49,6 +53,18 @@ public static void tearDown() {
             couchbase.stop();
         }
     }
+    
+    @Before
+    public void before() {
+        numberConnections = getNumConnections();
+    }
+
+    @After
+    public void after() {
+        // the connections active before must be equal to the connections active after
+        long numConnectionsAfter = getNumConnections();
+        assertEquals(numberConnections, numConnectionsAfter);
```

Comment:
what happens if not and what would be the reason for not?

Reply:
For example if I don't wrap correctly the procedure in a `try-res`, and so I don't call the `apoc.java.CouchbaseConnection.close()` method to disconnect the `Cluster` and shutdown the `ClusterEnvironment`, 
I receive a `numberConnections` equals to `numConnectionsAfter + 1`.
So, this assert is useful to verify that you have put the `try-res` wherever it's needed